compiler/AST/iterator.cpp

```diff
@@ -553,12 +551,10 @@ buildZip3(IteratorInfo* ii, Vec<BaseAST*>& asts, BlockStmt* singleLoop) {
                                         new CallExpr(PRIM_CAST, moreType->symbol, condExpr)));
     zip3body->insertAtTail(new CallExpr(PRIM_SET_MEMBER, ii->zip3->_this,
                                         ii->iclass->getField("more"), condTemp));
-  } else if (ForLoop* forLoop = toForLoop(singleLoop)) {
-    SymExpr* index = forLoop->indexGet()->copy(&map);
-
-    zip3body->insertAtTail(new CondStmt(index,
-                                        new CallExpr(PRIM_SET_MEMBER, ii->zip3->_this, ii->iclass->getField("more"), new_IntSymbol(1)),
-                                        new CallExpr(PRIM_SET_MEMBER, ii->zip3->_this, ii->iclass->getField("more"), new_IntSymbol(0))));
+  } else if (isCForLoop(singleLoop)) {
+    // Do nothing.
```

Comment:
Might say that the equivalent to this is handled in buildIncr for CForLoops instead of just saying "Do nothing"